oraichain/assetlist.json

```diff
@@ -50,6 +50,516 @@
+          "exponent": 6
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "orai1lplapmgqnelqn253stz6kmvm3ulgdaytn89a8mz9y85xq8wd684s6xl3lt",
+      "base": "cw20:orai1lplapmgqnelqn253stz6kmvm3ulgdaytn89a8mz9y85xq8wd684s6xl3lt",
+      "name": "Xoch",
+      "display": "xoch",
+      "symbol": "xOCH",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/och.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/och.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/och.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/och.svg"
+      }
+    },
+    {
+      "description": "The wETH token is the cw20 token for the Oraichain.",
+      "denom_units": [
+        {
+          "denom": "cw20:orai1dqa52a7hxxuv8ghe7q5v0s36ra0cthea960q2cukznleqhk0wpnshfegez",
+          "exponent": 0
+        },
+        {
+          "denom": "WETH",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "orai1dqa52a7hxxuv8ghe7q5v0s36ra0cthea960q2cukznleqhk0wpnshfegez",
+      "base": "cw20:orai1dqa52a7hxxuv8ghe7q5v0s36ra0cthea960q2cukznleqhk0wpnshfegez",
+      "name": "Weth",
+      "display": "weth",
+      "symbol": "wETH",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/eth-black.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/eth-black.svg"
+        },
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/eth-white.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/eth-white.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/eth-white.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/eth-white.svg"
+      }
+    },
+    {
+      "description": "The BTC token is the cw20 token for the Oraichain.",
+      "denom_units": [
+        {
+          "denom": "cw20:orai10g6frpysmdgw5tdqke47als6f97aqmr8s3cljsvjce4n5enjftcqtamzsd",
+          "exponent": 0
+        },
+        {
+          "denom": "BTC",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "orai10g6frpysmdgw5tdqke47als6f97aqmr8s3cljsvjce4n5enjftcqtamzsd",
+      "base": "cw20:orai10g6frpysmdgw5tdqke47als6f97aqmr8s3cljsvjce4n5enjftcqtamzsd",
+      "name": "Btc",
+      "display": "btc",
+      "symbol": "BTC",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/bitcoin.png",
```

Comment:
For all logos that aren't new, please don't upload duplicate images. Instead use image_sync and point back to the original (in this case /_non-cosmos/bitcoin/images/bitcoin.png)
Perhaps you can upload the svg and other new images to the directory of its origin instead?